hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/OMRequestTestUtils.java

```diff
@@ -531,7 +531,8 @@ public static void addBucketToDB(String volumeName, String bucketName,
 
   public static OzoneManagerProtocolProtos.OMRequest createBucketRequest(
       String bucketName, String volumeName, boolean isVersionEnabled,
-      OzoneManagerProtocolProtos.StorageTypeProto storageTypeProto) {
+      OzoneManagerProtocolProtos.StorageTypeProto storageTypeProto,
+      int clientVersion) {
```

Comment:
Is the `clientVersion` necessary for the tests, or can we just use the static from `ClientVersion`?